kafka/manager.go

```diff
@@ -197,11 +216,18 @@ func (m *Manager) MonitorConsumerLag(topicConsumers []apmqueue.TopicConsumer) (m
 				}
 				topic = topic[len(namespacePrefix):]
 
+				var matchesRegex bool
+				for _, re := range regex {
+					if l.Group == re.consumer && re.regex.MatchString(topic) {
```

Comment:
For educating myself: why do we need the Group to equal the consumer?

Reply:
The consumer is the consumer group. They're just named differently in the config vs the Kafka sdk